primitives/src/lib.rs

```diff
@@ -288,6 +288,10 @@ sp_api::decl_runtime_apis! {
         /// also as `aleph_key` - consensus engine's part of session keys) in the current session
         /// of AlephBFT (finalisation committee).
         fn key_owner(key: AuthorityId) -> Option<AccountId>;
+        /// Returns inflation from now to now + 1 year.
```

Comment:
Add "capped at 100%" or something like that - `Perbill` can only represent numbers not larger than one.

Reply:
done